packages/blocks/timer/src/app.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState, useCallback } from "react";
+import { BlockComponent } from "blockprotocol/react";
+
+import { TextField, Button } from "@mui/material";
```

Comment:
```
'Button' import from '@mui/material' is restricted. Please use the custom wrapper component in src/component instead
```

We are using this custom ESLint rule in `frontend`, so it’s kind of irrelevant here. Feel free to add

```
// eslint-disable-next-line no-restricted-imports --false-positive frontend-specific rule
```

Reply:
Oh sorry I made a typo. It was meant to be 

```
// eslint-disable-next-line no-restricted-imports -- false-positive frontend-specific rule
```

Happy for it to stay as it is. We try to add `-- explanation` in most cases to communicate our intent to other devs in the team.

Reply:
@kachkaev we shouldn't even have this route at the route, no? we have duplicate rules in frontend's eslint config, except they refer to the correct folder (`shared/ui`) – this rule is duplicative and refers to the wrong folder (`src/components`)

Reply:
Can this be quickly actioned, or otherwise captured in a `@todo` or ticket? ^ @kachkaev

Reply:
[Asana task](https://app.asana.com/0/0/1202221335010315/f) _(internal link)_